classes/various/extInteger.sc

```diff
@@ -0,0 +1,45 @@
+/*
+This file is part of MathLib, a Quark for SuperCollider:
+
+<https://github.com/supercollider-quarks/MathLib>
+
+You should have received a copy of the GNU General Public License v3 along with
+this distribution. If not, see:
+
+<https://www.gnu.org/licenses/gpl-3.0.txt>.
+*/
+
+
+//---------------------------------------------------------------------
+//	MathLib: Some mathematical extensions to SuperCollider.
+//
+// 	Extension: Integer
+//
+//---------------------------------------------------------------------
+
+
+/* J Anderson, 2019 */
++ Integer {
+
+	squareOf {
+		var sum = 0;
+		var res;
+
+		^if(this.isNegative, {
+			nil
+		}, {
+			res = 0;
+
+			while({ sum < this }, {
+				sum = Array.series(res + 1, 1, 2).sum;
+				res = res + 1;
```

Comment:
equivalent?
```supercollider
res = res + 1;
sum = Array.series(res, 1, 2).sum;
```

Reply:
better!

Reply:
But why the array? The sum of all numbers from 1 to N step 2 is I think something like `(N div: 2) squared`or so...

Reply:
I'll leave this to @joslloand who authored this, I just chimed in because I thought the nth root was an interesting problem and potentially a related solution or possible other addition to the library 😉

Reply:
Yes, thank you @mtmccrea !
The above can be replaced by 
```
res = res + 1;
sum = res.squared; // (1,3,..).sum 
```